include/sndfile.h

```diff
@@ -365,9 +365,9 @@ typedef	struct sf_private_tag	SNDFILE ;
 ** and the Microsoft compiler.
 */
 
-typedef @TYPEOF_SF_COUNT_T@	sf_count_t ;
+typedef int64_t			sf_count_t ;
 #ifndef SF_COUNT_MAX
-#define SF_COUNT_MAX		@SF_COUNT_MAX@
+#define SF_COUNT_MAX		0x7FFFFFFFFFFFFFFFLL
```

Comment:
@manxorist I thought a bit more, what about `INT64_MAX` here? That avoids the `long` vs `long long` literal discrepancy

Reply:
Yeah, that's probably a good additional cleanup. `LL`/`ll` suffix was what was always used before, I did just not consider changing it.

Do you want me to submit another pull request or will you just commit that yourself?

Reply:
I'm a bit busy, so if you did it, that would help me :slightly_smiling_face:

Reply:
I'm also busy right now, I'll do it tomorrow. 🙂

Reply:
found some spare time, see #870